Changed command error print to use proper System.err

Stupastian opened the pull request.

Commit:
Fix according to comments in Input validation for EasyCryptoConsole PR #32

Stupastian committed:
Changed command error print to use proper System.err

Amateur-S commented on JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java:
```diff
@@ -35,7 +35,7 @@ public static void main(String[] args) {
 					return;
 				} else {
-					console.printf("Sorry, '%s' is not a valid command.\n", d);
+					System.err.println("Sorry, '%s' is not a valid command.\n", d);
```
![image](https://user-images.githubusercontent.com/43349586/68947099-4f5e1280-07bd-11ea-832a-97685243e61c.png)
Let's see how adding pictures work. In short it won't build so can't accept this.

Amateur-S reviewed (changes requested):
Won't build so needs work.

Amateur-S committed:
Fixed syntax

Again fixed the syntax to be able to build the solution. Also tested and works perfectly.

Amateur-S commented:
Added fix as commit
